src/vse_sync_pp/analyze.py

```diff
@@ -0,0 +1,68 @@
+### SPDX-License-Identifier: GPL-2.0-or-later
+import sys
+import json
+import pandas as pd
+from argparse import ArgumentParser
+
+from .common import open_input
+
+from .parse import PARSERS
+from .analyzers import ppsdpll
+from . plot import Plotter
+
+ANALYZERS = {
+    cls.id_: cls for cls in (
+        ppsdpll.PhaseOffsetTimeError,
+        ppsdpll.ConstantPhaseOffsetTimeError,
+        ppsdpll.PhaseOffsetTimeDeviation,
+        ppsdpll.MaxPhaseOffsetTimeIntervalError
+    )
+}
+def analyzer2parser(analyzer_id):
```

Comment:
An analyzer's dependency is upon the canonical format it expects. Would be better to put the parser id (which produces that canonical format) in the analyzer class?